src/main/resources/integration-jira-log-messages.properties

```diff
@@ -0,0 +1,4 @@
+integration.jira.url.base.invalid=Invalid base URL.
```

Comment:
I believe you should change this message to show which base URL was configured.

Something like that:
integration.jira.url.base.invalid=Invalid JIRA base URL. Base URL configured: {0}